src/main/resources/config/application-dev.yml

```diff
@@ -2,7 +2,7 @@ spring:
   jackson:
       serialization.indent_output: true
   datasource:
-    url: jdbc:mysql://${DATABASE_ADDRESS:localhost}/${DATABASE_NAME:faf}?useSSL=false
+    url: jdbc:mysql://${DATABASE_ADDRESS:192.168.99.100}/${DATABASE_NAME:faf}?useSSL=false
```

Comment:
Euhm, you are too fast, I force pushed that exact second you created the review..... ;)